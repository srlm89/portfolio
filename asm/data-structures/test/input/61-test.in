ERASE actual/61-test.out
INSERT 35 2.667740 Butterfly on a Wheel.2007   720p
PRINT_IN_ORDER actual/61-test.out
INSERT 34 9.621030 James Bond 05 - You Only Live Twice [1967]
INSERT 2 9.582525 Interview with the Vampire   720p
REMOVE_ID 2
PRINT_IN_ORDER actual/61-test.out
INSERT 46 8.178794 James Bond 19 - The World is not Enough [1999] [720p]
INSERT 7 5.414444 La Lingerie [BluRay] [720p] [DTS]
INSERT 4 4.435619 Bride Wars   720p
REMOVE_ID 46
PRINT_IN_ORDER actual/61-test.out
INSERT 45 4.573367 catch me if u can - 720p
INSERT 55 8.694502 Fearless - 720p
INSERT 36 6.509718 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs
INSERT 49 3.396795 Miss March [720p]
REMOVE_ID 7
REMOVE_ID 55
PRINT_IN_ORDER actual/61-test.out
INSERT 62 0.919147 Surrogates [2009] [720p]
INSERT 38 0.229865 Shinjuku Incident
INSERT 5 3.141994 Kung Fu Panda Secrets of the Furious Five    720p
INSERT 30 8.430176 Connected   720p
INSERT 6 7.968957 Jumanji [720p]
REMOVE_ID 45
REMOVE_ID 30
PRINT_IN_ORDER actual/61-test.out
INSERT 47 9.105373 PIXAR Movie
INSERT 27 8.005890 Finding Nemo [2003] [720p]
INSERT 66 2.668123 Jumanji [720p]
INSERT 11 0.657490 District 9 [2009] [720p]
INSERT 75 5.522857 Apollo 13   1080p
INSERT 8 4.217069 Incredibles, The 1080p
REMOVE_ID 36
REMOVE_ID 6
REMOVE_ID 66
PRINT_IN_ORDER actual/61-test.out
INSERT 31 0.107595 Click   1080p
INSERT 50 1.732509 Cloverfield   1080p
INSERT 79 9.026637 Star Wars Saga  Collection 720p
INSERT 67 6.398224 American Psycho   720p
INSERT 16 1.086566 Watchmen [720p]
INSERT 83 8.029873 Lord of war   720p
INSERT 32 2.661844 SAW III   720p
REMOVE_ID 32
REMOVE_ID 8
REMOVE_ID 79
PRINT_IN_ORDER actual/61-test.out
INSERT 56 2.308981 Hellboy 1    1080p
INSERT 13 6.058532 007.1983.Octopussy.720p.hdtv.x264
INSERT 86 8.852988 Beauty And The Beast    720p
INSERT 28 8.549820 Van.Helsing  1080p
INSERT 3 3.649883 James Bond 22 - Quantum of Solace [2008] [720p]
INSERT 39 1.595621 Spiderman 1   1080p
INSERT 59 1.531880 Poseidon   1080p
INSERT 33 3.010539 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
REMOVE_ID 49
REMOVE_ID 47
REMOVE_ID 35
REMOVE_ID 3
PRINT_IN_ORDER actual/61-test.out
INSERT 89 3.025610 X-Men 1    720p
INSERT 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD
INSERT 9 8.682039 Lord Of The Ring  - The Two Tower   720p
INSERT 90 8.286174 Armageddon [1998] [720p]
INSERT 63 8.085624 Gone In 60 Seconds    1080p
INSERT 24 6.716611 James Bond 18 - Tommorow Never Dies [1997] [720p]
INSERT 57 9.132495 Con Air   720p
INSERT 12 7.422127 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 94 7.743565 Jeniffer's Body ( 2009)    1080p
REMOVE_ID 28
REMOVE_ID 33
REMOVE_ID 50
REMOVE_ID 62
PRINT_IN_ORDER actual/61-test.out
INSERT 48 1.647311 contents.txt
INSERT 41 9.123050 Push    1080p
INSERT 72 4.997138 Iron Man    1080p
INSERT 54 3.469934 Night At The Museum 2 [720p]
INSERT 37 6.759244 Erin Brockovich 2000.720p
INSERT 74 2.674136 All About Steve 720P
INSERT 51 2.422542 Final Destination
INSERT 96 8.761239 Elite Squad 1080p
INSERT 52 2.115735 Bee Movie   720p
INSERT 10 6.461680 Ice Age 2 The Meltdown [2006] [720p]
REMOVE_ID 5
REMOVE_ID 90
REMOVE_ID 89
REMOVE_ID 41
REMOVE_ID 86
PRINT_IN_ORDER actual/61-test.out
INSERT 42 3.832363 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN]
INSERT 65 3.083252 300   720p
INSERT 106 9.369156 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE
INSERT 64 7.978053 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS
INSERT 80 3.406531 Pinocchio [1940] [720p]
INSERT 118 2.758996 Elite Squad 1080p
INSERT 98 7.231980 Direct Contact    720p
INSERT 112 4.090403 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 19 9.847323 Robocop Collection   720p
INSERT 95 1.285675 Harold & Kumar   720p
INSERT 100 6.995941 Toy.Story.1995.720p.HDTV.x264-CtrlHD
REMOVE_ID 56
REMOVE_ID 13
REMOVE_ID 38
REMOVE_ID 54
REMOVE_ID 52
PRINT_IN_ORDER actual/61-test.out
INSERT 53 6.304341 G.I.Joe The Rise of Cobra [2009] [720p]
INSERT 125 6.501030 Back.to.the.Future.Part.II.1989.720p.HDTV.x264
INSERT 82 3.216636 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p]
INSERT 73 5.733015 Spiderman 3   1080p
INSERT 130 4.067507 James Bond 21 - Casino Royale [2006]
INSERT 129 7.851270 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC
INSERT 116 4.174704 Night At The Museum 2 [720p]
INSERT 22 2.840793 Sex drive   720p
INSERT 78 4.749121 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p
INSERT 85 5.612964 Spiderman 2   1080p
INSERT 113 6.454141 Romeo And Juliet   720p
INSERT 26 6.706576 The Dark Knight [2008]   1080p
REMOVE_ID 12
REMOVE_ID 82
REMOVE_ID 85
REMOVE_ID 24
REMOVE_ID 57
REMOVE_ID 94
PRINT_IN_ORDER actual/61-test.out
INSERT 111 1.440400 Top.Gun.1986.720p.HDDVD.x264-SEPTiC
INSERT 103 5.361904 XxX.2002.BDRiP.720p.AC3.5.1.READnfo.x264-DEFiNiTiON
INSERT 137 7.548032 Batman Dark Knight   1080p
INSERT 14 8.026560 Final Fantasy The Spirits Within    720p
INSERT 44 6.113136 SAW V
INSERT 21 5.618464 Horseman    720p
INSERT 69 8.363192 Home Alone 1 & 2   720p
INSERT 134 5.346753 Toy.Story.1995.720p.HDTV.x264-CtrlHD
INSERT 143 0.582350 How To Lose Friends & Alienate People   1080p
INSERT 18 9.326180 Daredevil - 720p
INSERT 126 7.435433 I Am Legend    720p
INSERT 122 6.392681 Batman Begins   1080p
INSERT 71 9.312600 The Bank Job [720p]
REMOVE_ID 19
REMOVE_ID 111
REMOVE_ID 22
REMOVE_ID 16
REMOVE_ID 31
REMOVE_ID 72
PRINT_IN_ORDER actual/61-test.out
INSERT 23 5.907140 Hellboy 2     1080p
INSERT 70 5.403942 Lawrence of Arabia   1080p
INSERT 119 7.926245 Jumper    1080p
INSERT 93 1.195409 Taken 1080p
INSERT 0 0.050955 Red Dragon    1080p
INSERT 117 7.637623 The Matrix [1999] [1080p]
INSERT 152 5.157984 Batman Cartoon - Gotham Knight   720p
INSERT 132 1.423342 The Dark Knight [2008]   1080p
INSERT 17 0.261317 Romeo And Juliet   720p
INSERT 133 0.770863 Babel.720p
INSERT 127 9.613341 Armageddon [1998] [720p]
INSERT 140 6.595464 Incredibles, The 1080p
INSERT 1 8.335478 Get Smart (2008) 1080p
INSERT 87 5.075147 Mission Impossible Trilogy   720p
REMOVE_ID 152
REMOVE_ID 106
REMOVE_ID 75
REMOVE_ID 73
REMOVE_ID 14
REMOVE_ID 63
REMOVE_ID 125
PRINT_IN_ORDER actual/61-test.out
INSERT 43 0.533172 Shutter   720p
INSERT 105 6.498734 catch me if u can - 720p
INSERT 136 3.316881 X-Men 1    720p
INSERT 157 3.225240 Crank 2, High Voltage   1080p
INSERT 170 9.863478 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs
INSERT 164 1.441499 Baby on Board (2009)    720p
INSERT 141 6.307341 The Sniper 720p
INSERT 115 5.125791 Rocky 1 [1976]  [BluRay] [720p] [AC3]
INSERT 156 7.831132 Braveheart 720p
INSERT 151 3.168911 SWAT    720p
INSERT 144 8.986089 American Psycho   720p
INSERT 99 4.211498 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p]
INSERT 155 3.274182 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS
INSERT 84 7.269444 X-Men 1    720p
INSERT 20 5.233262 Watchmen [720p]
REMOVE_ID 10
REMOVE_ID 80
REMOVE_ID 26
REMOVE_ID 117
REMOVE_ID 70
REMOVE_ID 39
REMOVE_ID 64
PRINT_IN_ORDER actual/61-test.out
