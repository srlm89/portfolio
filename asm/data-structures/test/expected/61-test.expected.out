[ 35 2.667740 Butterfly on a Wheel.2007   720p ]
[ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 35 2.667740 Butterfly on a Wheel.2007   720p ]
[ 4 4.435619 Bride Wars   720p ][ 7 5.414444 La Lingerie [BluRay] [720p] [DTS] ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 35 2.667740 Butterfly on a Wheel.2007   720p ]
[ 4 4.435619 Bride Wars   720p ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 35 2.667740 Butterfly on a Wheel.2007   720p ][ 36 6.509718 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 45 4.573367 catch me if u can - 720p ][ 49 3.396795 Miss March [720p] ]
[ 4 4.435619 Bride Wars   720p ][ 5 3.141994 Kung Fu Panda Secrets of the Furious Five    720p ][ 6 7.968957 Jumanji [720p] ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 35 2.667740 Butterfly on a Wheel.2007   720p ][ 36 6.509718 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 38 0.229865 Shinjuku Incident ][ 49 3.396795 Miss March [720p] ][ 62 0.919147 Surrogates [2009] [720p] ]
[ 4 4.435619 Bride Wars   720p ][ 5 3.141994 Kung Fu Panda Secrets of the Furious Five    720p ][ 8 4.217069 Incredibles, The 1080p ][ 11 0.657490 District 9 [2009] [720p] ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 35 2.667740 Butterfly on a Wheel.2007   720p ][ 38 0.229865 Shinjuku Incident ][ 47 9.105373 PIXAR Movie ][ 49 3.396795 Miss March [720p] ][ 62 0.919147 Surrogates [2009] [720p] ][ 75 5.522857 Apollo 13   1080p ]
[ 4 4.435619 Bride Wars   720p ][ 5 3.141994 Kung Fu Panda Secrets of the Furious Five    720p ][ 11 0.657490 District 9 [2009] [720p] ][ 16 1.086566 Watchmen [720p] ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 31 0.107595 Click   1080p ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 35 2.667740 Butterfly on a Wheel.2007   720p ][ 38 0.229865 Shinjuku Incident ][ 47 9.105373 PIXAR Movie ][ 49 3.396795 Miss March [720p] ][ 50 1.732509 Cloverfield   1080p ][ 62 0.919147 Surrogates [2009] [720p] ][ 67 6.398224 American Psycho   720p ][ 75 5.522857 Apollo 13   1080p ][ 83 8.029873 Lord of war   720p ]
[ 4 4.435619 Bride Wars   720p ][ 5 3.141994 Kung Fu Panda Secrets of the Furious Five    720p ][ 11 0.657490 District 9 [2009] [720p] ][ 13 6.058532 007.1983.Octopussy.720p.hdtv.x264 ][ 16 1.086566 Watchmen [720p] ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 28 8.549820 Van.Helsing  1080p ][ 31 0.107595 Click   1080p ][ 33 3.010539 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 38 0.229865 Shinjuku Incident ][ 39 1.595621 Spiderman 1   1080p ][ 50 1.732509 Cloverfield   1080p ][ 56 2.308981 Hellboy 1    1080p ][ 59 1.531880 Poseidon   1080p ][ 62 0.919147 Surrogates [2009] [720p] ][ 67 6.398224 American Psycho   720p ][ 75 5.522857 Apollo 13   1080p ][ 83 8.029873 Lord of war   720p ][ 86 8.852988 Beauty And The Beast    720p ]
[ 4 4.435619 Bride Wars   720p ][ 5 3.141994 Kung Fu Panda Secrets of the Furious Five    720p ][ 9 8.682039 Lord Of The Ring  - The Two Tower   720p ][ 11 0.657490 District 9 [2009] [720p] ][ 12 7.422127 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 13 6.058532 007.1983.Octopussy.720p.hdtv.x264 ][ 16 1.086566 Watchmen [720p] ][ 24 6.716611 James Bond 18 - Tommorow Never Dies [1997] [720p] ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 31 0.107595 Click   1080p ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 38 0.229865 Shinjuku Incident ][ 39 1.595621 Spiderman 1   1080p ][ 56 2.308981 Hellboy 1    1080p ][ 57 9.132495 Con Air   720p ][ 59 1.531880 Poseidon   1080p ][ 63 8.085624 Gone In 60 Seconds    1080p ][ 67 6.398224 American Psycho   720p ][ 75 5.522857 Apollo 13   1080p ][ 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 83 8.029873 Lord of war   720p ][ 86 8.852988 Beauty And The Beast    720p ][ 89 3.025610 X-Men 1    720p ][ 90 8.286174 Armageddon [1998] [720p] ][ 94 7.743565 Jeniffer's Body ( 2009)    1080p ]
[ 4 4.435619 Bride Wars   720p ][ 9 8.682039 Lord Of The Ring  - The Two Tower   720p ][ 10 6.461680 Ice Age 2 The Meltdown [2006] [720p] ][ 11 0.657490 District 9 [2009] [720p] ][ 12 7.422127 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 13 6.058532 007.1983.Octopussy.720p.hdtv.x264 ][ 16 1.086566 Watchmen [720p] ][ 24 6.716611 James Bond 18 - Tommorow Never Dies [1997] [720p] ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 31 0.107595 Click   1080p ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 37 6.759244 Erin Brockovich 2000.720p ][ 38 0.229865 Shinjuku Incident ][ 39 1.595621 Spiderman 1   1080p ][ 48 1.647311 contents.txt ][ 51 2.422542 Final Destination ][ 52 2.115735 Bee Movie   720p ][ 54 3.469934 Night At The Museum 2 [720p] ][ 56 2.308981 Hellboy 1    1080p ][ 57 9.132495 Con Air   720p ][ 59 1.531880 Poseidon   1080p ][ 63 8.085624 Gone In 60 Seconds    1080p ][ 67 6.398224 American Psycho   720p ][ 72 4.997138 Iron Man    1080p ][ 74 2.674136 All About Steve 720P ][ 75 5.522857 Apollo 13   1080p ][ 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 83 8.029873 Lord of war   720p ][ 94 7.743565 Jeniffer's Body ( 2009)    1080p ][ 96 8.761239 Elite Squad 1080p ]
[ 4 4.435619 Bride Wars   720p ][ 9 8.682039 Lord Of The Ring  - The Two Tower   720p ][ 10 6.461680 Ice Age 2 The Meltdown [2006] [720p] ][ 11 0.657490 District 9 [2009] [720p] ][ 12 7.422127 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 16 1.086566 Watchmen [720p] ][ 19 9.847323 Robocop Collection   720p ][ 24 6.716611 James Bond 18 - Tommorow Never Dies [1997] [720p] ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 31 0.107595 Click   1080p ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 37 6.759244 Erin Brockovich 2000.720p ][ 39 1.595621 Spiderman 1   1080p ][ 42 3.832363 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 48 1.647311 contents.txt ][ 51 2.422542 Final Destination ][ 57 9.132495 Con Air   720p ][ 59 1.531880 Poseidon   1080p ][ 63 8.085624 Gone In 60 Seconds    1080p ][ 64 7.978053 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 65 3.083252 300   720p ][ 67 6.398224 American Psycho   720p ][ 72 4.997138 Iron Man    1080p ][ 74 2.674136 All About Steve 720P ][ 75 5.522857 Apollo 13   1080p ][ 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 80 3.406531 Pinocchio [1940] [720p] ][ 83 8.029873 Lord of war   720p ][ 94 7.743565 Jeniffer's Body ( 2009)    1080p ][ 95 1.285675 Harold & Kumar   720p ][ 96 8.761239 Elite Squad 1080p ][ 98 7.231980 Direct Contact    720p ][ 100 6.995941 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 106 9.369156 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 112 4.090403 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 118 2.758996 Elite Squad 1080p ]
[ 4 4.435619 Bride Wars   720p ][ 9 8.682039 Lord Of The Ring  - The Two Tower   720p ][ 10 6.461680 Ice Age 2 The Meltdown [2006] [720p] ][ 11 0.657490 District 9 [2009] [720p] ][ 16 1.086566 Watchmen [720p] ][ 19 9.847323 Robocop Collection   720p ][ 22 2.840793 Sex drive   720p ][ 26 6.706576 The Dark Knight [2008]   1080p ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 31 0.107595 Click   1080p ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 37 6.759244 Erin Brockovich 2000.720p ][ 39 1.595621 Spiderman 1   1080p ][ 42 3.832363 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 48 1.647311 contents.txt ][ 51 2.422542 Final Destination ][ 53 6.304341 G.I.Joe The Rise of Cobra [2009] [720p] ][ 59 1.531880 Poseidon   1080p ][ 63 8.085624 Gone In 60 Seconds    1080p ][ 64 7.978053 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 65 3.083252 300   720p ][ 67 6.398224 American Psycho   720p ][ 72 4.997138 Iron Man    1080p ][ 73 5.733015 Spiderman 3   1080p ][ 74 2.674136 All About Steve 720P ][ 75 5.522857 Apollo 13   1080p ][ 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 78 4.749121 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 80 3.406531 Pinocchio [1940] [720p] ][ 83 8.029873 Lord of war   720p ][ 95 1.285675 Harold & Kumar   720p ][ 96 8.761239 Elite Squad 1080p ][ 98 7.231980 Direct Contact    720p ][ 100 6.995941 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 106 9.369156 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 112 4.090403 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 113 6.454141 Romeo And Juliet   720p ][ 116 4.174704 Night At The Museum 2 [720p] ][ 118 2.758996 Elite Squad 1080p ][ 125 6.501030 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 129 7.851270 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC ][ 130 4.067507 James Bond 21 - Casino Royale [2006] ]
[ 4 4.435619 Bride Wars   720p ][ 9 8.682039 Lord Of The Ring  - The Two Tower   720p ][ 10 6.461680 Ice Age 2 The Meltdown [2006] [720p] ][ 11 0.657490 District 9 [2009] [720p] ][ 14 8.026560 Final Fantasy The Spirits Within    720p ][ 18 9.326180 Daredevil - 720p ][ 21 5.618464 Horseman    720p ][ 26 6.706576 The Dark Knight [2008]   1080p ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 37 6.759244 Erin Brockovich 2000.720p ][ 39 1.595621 Spiderman 1   1080p ][ 42 3.832363 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 44 6.113136 SAW V ][ 48 1.647311 contents.txt ][ 51 2.422542 Final Destination ][ 53 6.304341 G.I.Joe The Rise of Cobra [2009] [720p] ][ 59 1.531880 Poseidon   1080p ][ 63 8.085624 Gone In 60 Seconds    1080p ][ 64 7.978053 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 65 3.083252 300   720p ][ 67 6.398224 American Psycho   720p ][ 69 8.363192 Home Alone 1 & 2   720p ][ 71 9.312600 The Bank Job [720p] ][ 73 5.733015 Spiderman 3   1080p ][ 74 2.674136 All About Steve 720P ][ 75 5.522857 Apollo 13   1080p ][ 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 78 4.749121 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 80 3.406531 Pinocchio [1940] [720p] ][ 83 8.029873 Lord of war   720p ][ 95 1.285675 Harold & Kumar   720p ][ 96 8.761239 Elite Squad 1080p ][ 98 7.231980 Direct Contact    720p ][ 100 6.995941 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 103 5.361904 XxX.2002.BDRiP.720p.AC3.5.1.READnfo.x264-DEFiNiTiON ][ 106 9.369156 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 112 4.090403 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 113 6.454141 Romeo And Juliet   720p ][ 116 4.174704 Night At The Museum 2 [720p] ][ 118 2.758996 Elite Squad 1080p ][ 122 6.392681 Batman Begins   1080p ][ 125 6.501030 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 126 7.435433 I Am Legend    720p ][ 129 7.851270 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC ][ 130 4.067507 James Bond 21 - Casino Royale [2006] ][ 134 5.346753 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 137 7.548032 Batman Dark Knight   1080p ][ 143 0.582350 How To Lose Friends & Alienate People   1080p ]
[ 0 0.050955 Red Dragon    1080p ][ 1 8.335478 Get Smart (2008) 1080p ][ 4 4.435619 Bride Wars   720p ][ 9 8.682039 Lord Of The Ring  - The Two Tower   720p ][ 10 6.461680 Ice Age 2 The Meltdown [2006] [720p] ][ 11 0.657490 District 9 [2009] [720p] ][ 17 0.261317 Romeo And Juliet   720p ][ 18 9.326180 Daredevil - 720p ][ 21 5.618464 Horseman    720p ][ 23 5.907140 Hellboy 2     1080p ][ 26 6.706576 The Dark Knight [2008]   1080p ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 37 6.759244 Erin Brockovich 2000.720p ][ 39 1.595621 Spiderman 1   1080p ][ 42 3.832363 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 44 6.113136 SAW V ][ 48 1.647311 contents.txt ][ 51 2.422542 Final Destination ][ 53 6.304341 G.I.Joe The Rise of Cobra [2009] [720p] ][ 59 1.531880 Poseidon   1080p ][ 64 7.978053 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 65 3.083252 300   720p ][ 67 6.398224 American Psycho   720p ][ 69 8.363192 Home Alone 1 & 2   720p ][ 70 5.403942 Lawrence of Arabia   1080p ][ 71 9.312600 The Bank Job [720p] ][ 74 2.674136 All About Steve 720P ][ 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 78 4.749121 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 80 3.406531 Pinocchio [1940] [720p] ][ 83 8.029873 Lord of war   720p ][ 87 5.075147 Mission Impossible Trilogy   720p ][ 93 1.195409 Taken 1080p ][ 95 1.285675 Harold & Kumar   720p ][ 96 8.761239 Elite Squad 1080p ][ 98 7.231980 Direct Contact    720p ][ 100 6.995941 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 103 5.361904 XxX.2002.BDRiP.720p.AC3.5.1.READnfo.x264-DEFiNiTiON ][ 112 4.090403 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 113 6.454141 Romeo And Juliet   720p ][ 116 4.174704 Night At The Museum 2 [720p] ][ 117 7.637623 The Matrix [1999] [1080p] ][ 118 2.758996 Elite Squad 1080p ][ 119 7.926245 Jumper    1080p ][ 122 6.392681 Batman Begins   1080p ][ 126 7.435433 I Am Legend    720p ][ 127 9.613341 Armageddon [1998] [720p] ][ 129 7.851270 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC ][ 130 4.067507 James Bond 21 - Casino Royale [2006] ][ 132 1.423342 The Dark Knight [2008]   1080p ][ 133 0.770863 Babel.720p ][ 134 5.346753 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 137 7.548032 Batman Dark Knight   1080p ][ 140 6.595464 Incredibles, The 1080p ][ 143 0.582350 How To Lose Friends & Alienate People   1080p ]
[ 0 0.050955 Red Dragon    1080p ][ 1 8.335478 Get Smart (2008) 1080p ][ 4 4.435619 Bride Wars   720p ][ 9 8.682039 Lord Of The Ring  - The Two Tower   720p ][ 11 0.657490 District 9 [2009] [720p] ][ 17 0.261317 Romeo And Juliet   720p ][ 18 9.326180 Daredevil - 720p ][ 20 5.233262 Watchmen [720p] ][ 21 5.618464 Horseman    720p ][ 23 5.907140 Hellboy 2     1080p ][ 27 8.005890 Finding Nemo [2003] [720p] ][ 34 9.621030 James Bond 05 - You Only Live Twice [1967] ][ 37 6.759244 Erin Brockovich 2000.720p ][ 42 3.832363 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 43 0.533172 Shutter   720p ][ 44 6.113136 SAW V ][ 48 1.647311 contents.txt ][ 51 2.422542 Final Destination ][ 53 6.304341 G.I.Joe The Rise of Cobra [2009] [720p] ][ 59 1.531880 Poseidon   1080p ][ 65 3.083252 300   720p ][ 67 6.398224 American Psycho   720p ][ 69 8.363192 Home Alone 1 & 2   720p ][ 71 9.312600 The Bank Job [720p] ][ 74 2.674136 All About Steve 720P ][ 77 7.872728 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 78 4.749121 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 83 8.029873 Lord of war   720p ][ 84 7.269444 X-Men 1    720p ][ 87 5.075147 Mission Impossible Trilogy   720p ][ 93 1.195409 Taken 1080p ][ 95 1.285675 Harold & Kumar   720p ][ 96 8.761239 Elite Squad 1080p ][ 98 7.231980 Direct Contact    720p ][ 99 4.211498 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 100 6.995941 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 103 5.361904 XxX.2002.BDRiP.720p.AC3.5.1.READnfo.x264-DEFiNiTiON ][ 105 6.498734 catch me if u can - 720p ][ 112 4.090403 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 113 6.454141 Romeo And Juliet   720p ][ 115 5.125791 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 116 4.174704 Night At The Museum 2 [720p] ][ 118 2.758996 Elite Squad 1080p ][ 119 7.926245 Jumper    1080p ][ 122 6.392681 Batman Begins   1080p ][ 126 7.435433 I Am Legend    720p ][ 127 9.613341 Armageddon [1998] [720p] ][ 129 7.851270 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC ][ 130 4.067507 James Bond 21 - Casino Royale [2006] ][ 132 1.423342 The Dark Knight [2008]   1080p ][ 133 0.770863 Babel.720p ][ 134 5.346753 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 136 3.316881 X-Men 1    720p ][ 137 7.548032 Batman Dark Knight   1080p ][ 140 6.595464 Incredibles, The 1080p ][ 141 6.307341 The Sniper 720p ][ 143 0.582350 How To Lose Friends & Alienate People   1080p ][ 144 8.986089 American Psycho   720p ][ 151 3.168911 SWAT    720p ][ 155 3.274182 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 156 7.831132 Braveheart 720p ][ 157 3.225240 Crank 2, High Voltage   1080p ][ 164 1.441499 Baby on Board (2009)    720p ][ 170 9.863478 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ]
